invenio_rdm_records/fixtures/data/vocabularies/resource_types.yaml

```diff
@@ -574,6 +718,24 @@
   tags:
     - depositable
     - linkable
+- id: computationalnotebook
+  icon: code
+  props:
+    csl: software
+    datacite_general: Software
+    datacite_type: Computational notebook
+    openaire_resourceType: '40'
+    openaire_type: software
+    eurepo: info:eu-repo/semantics/other
+    schema.org: https://schema.org/SoftwareSourceCode
+    subtype: computational notebook
+    type: software
+  title:
+    en: Computational notebook
+    de: Computer-Notizbuch
```

Comment:
`de: Computational Notebook`

additional information for translation:
datacite description: A virtual notebook environment used for literate programming